plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java

```diff
@@ -129,4 +134,9 @@ public static ColumnMappingMode getColumnMappingMode(Map<String, Object> tablePr
     {
         return ColumnMappingMode.valueOf(tableProperties.get(COLUMN_MAPPING_MODE_PROPERTY).toString().toUpperCase(ENGLISH));
     }
+
+    public static boolean getDeletionVectorsEnabled(Map<String, Object> tableProperties)
+    {
+        return (boolean) tableProperties.getOrDefault(DELETION_VECTORS_ENABLED_PROPERTY, false);
```

Comment:
Can we pick up the default from a catalog property ? That way admins can enable it for all new tables without requiring users to set it in every table definition.

Reply:
Added `delta.deletion-vectors-enabled` config property.